libs/utils/src/votes.ts

```diff
@@ -84,3 +88,28 @@ export const buildVoteFromCvr = ({
 
   return vote
 }
+
+/**
+ * Gets all the vote options a voter can make for a given yes/no contest.
+ * @returns ContestVoteOption[] ex. ['yes', 'no']
+ */
+export function getContestVoteOptionsForYesNoContest(
+  contest: YesNoContest
+): readonly YesNoVoteID[] {
+  assert(contest.type === 'yesno')
+  return ['yes', 'no']
+}
+
+/**
+ * Gets all the vote options a voter can make for a given contest. If write ins are allowed a single write in candidate ID is included.
+ * @returns ContestVoteOption[] ex. ['yes', 'no'] or ['aaron', 'bob', '__write-in']
+ */
+export function getContestVoteOptionsForCandidateContest(
+  contest: CandidateContest
+): readonly Candidate[] {
```

Comment:
We should have a larger discussion about our approach to immutability than makes sense to have in unrelated PR comments. However, I've been avoiding `readonly` on _return values_ to indicate that the caller now owns the value and can do what it likes with it. If you were returning a _shared_ value then keeping `readonly` would make sense.

Reply:
The object is readonly here though so I get a type error if I don't mark it as readonly.